PWGCF/Flow/Tasks/FlowPbPbTask.cxx

```diff
@@ -305,11 +372,16 @@ struct FlowPbPbTask {
     for (auto& track : tracks) {
       double pt = track.pt();
       double eta = track.eta();
+      double phi = track.phi();
```

Comment:
For the next iteration please, don't store the values unless you need them and don't convert them to `double`, they are stored as `float` in the tables
Don't be worried by repeating, for instance `track.pt()` several times. The compiler will take care of feasible optimizations